### PR TITLE
Feature/2: added licence, contributing, updated getting started, …

### PR DESCRIPTION
**Pull request by karolinayanko**
added open source license
added basic contributing description
updated README file and Getting Started section.

### CODE REVIEW

**Review by hromogen — APPROVED** [review]

**Review by sfounder — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Contributing
+We love pull requests from everyone. By participating in this project, you
```
Comment (kucherenko) [inline CONTRIBUTING.md:2]: ❤️

File: README.md [hunk 2]
```diff
@@ -2,40 +2,49 @@
...
+/node_modules    // dependecies shared by the packages
+/packages
+  /web           // web code [React]
+  /mobile        // mobile code [React Native]
+  /server        // server code [Node]
+  /core          // logic to be shared between web/mobile/server
+```
+
+## Contributing
+If you'd like to contribute, please follow this [recommendations](CONTRIBUTING.md).
```
Comment (oleksiydubovykTR) [inline README.md:47]: I'd recommend using "`these` recommendations ", since it's plural

File: README.md [hunk 3]
```diff
@@ -2,40 +2,49 @@
...
 
 If you need to add a dependency to a package, run `yarn add` in the package directory:
 ```
 [/packages/web]> yarn add redux
 ```
 
-## Run
-Use root or package level commands to run web:
+### Run
+Use root or package level commands to run specific package:
```
Comment (oleksiydubovykTR) [inline README.md:21]: The example was a specific one (web) followed by another specific one (mobile), since you left the second one, maybe rephrase this one like so:

Use root or package level commands to run a specific package, e.g. `web`:

**Review by oleksiydubovykTR — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (dubbha) [inline README.md]: The example was a specific one (web) followed by another specific one (mobile), since you left the second one, maybe rephrase this one like so:

Use root or package level commands to run a specific package, e.g. web:

File: README.md (hunk 2 above)
Comment (dubbha) [inline README.md]: I'd recommend using "these recommendations ", since it's plural :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+agree with out [license](https://opensource.org/licenses/MIT).
+
+#### Acceptable pull request must have:
+* new parts are covered with tests;
+* commit message with meaningful description;
+* code must follow our style guides.
```
Comment (dubbha) [inline CONTRIBUTING.md]: Please  also encorporate the following parts:

### Pull Request rules

 - At least two approving reviews required for PRs/
 - All checks must pass before merging.
 - Prepend your PR title with "WIP:" if you don't want it to be merged just yet
 - Squash/rebase to develop branch, keep the history linear.
 - Merge to higher branches.
 
### Code Review rules

 - Take your time to review but don't leave PRs hanging for too long.
 - You are on the same side. Share opinions, links, and perspectives.
 - Be nice. Discuss the code, not the author. Foster positive review culture.
 - You are not your code. Try not to get defensive.

### Commit messages

Try to follow the [conventional commits](https://www.conventionalcommits.org/) convention in your commit messages.
Incorporate the Trello task number in your commits so that it can be tracked back to the initial issue, e.g.:
```
git commit -m "feat(42): add deep thought dependency"
```
This will not be enforced in a precommit hook though, we're not developing a library, nor using release scripts analysing commit messages.

File: README.md (hunk 3 above)
Comment (karolinayanko) [reply README.md]: Done

File: CONTRIBUTING.md (hunk 4 above)
Comment (karolinayanko) [reply CONTRIBUTING.md]: Added to our new Wiki https://github.com/dubbha/croton/wiki

**Review by belgithub — APPROVED** [review]

**Review by oleksiydubovykTR — APPROVED** [review]

**Review by dubbha — APPROVED** [review]
